### PR TITLE
Fix API Approvals

### PR DESCRIPTION
**Pull request by SeanFeldman**

### CODE REVIEW

File: src/ServiceControl.UnitTests/ServiceControl.UnitTests.csproj [hunk 1]
```diff
@@ -27,9 +27,9 @@
     <PackageReference Include="NUnit3TestAdapter" Version="3.17.0" />
     <PackageReference Include="Particular.Approvals" Version="0.2.0" />
     <PackageReference Include="Particular.CodeRules" Version="0.7.0" />
-    <PackageReference Include="PublicApiGenerator" Version="9.3.0" />
     <PackageReference Include="RavenDB.Tests.Helpers" Version="3.5.10-patch-35303" />
     <PackageReference Include="System.ValueTuple" Version="4.5.0" />
+    <PackageReference Include="PublicApiGenerator" Version="10.2.0" />
```
Comment (bording) [inline src/ServiceControl.UnitTests/ServiceControl.UnitTests.csproj:32]: Why did this change lines?

Comment (SeanFeldman) [reply src/ServiceControl.UnitTests/ServiceControl.UnitTests.csproj]: Fixed

**Review by bording — APPROVED** [review]

File: src/ServiceControl.UnitTests/API/APIApprovals.cs [hunk 2]
```diff
@@ -24,14 +24,20 @@ class APIApprovals
         [Test]
         public void PublicClr()
         {
-            var publicApi = ApiGenerator.GeneratePublicApi(typeof(Bootstrapper).Assembly);
+            var publicApi = typeof(Bootstrapper).Assembly.GeneratePublicApi(new ApiGeneratorOptions
+            {
+                ExcludeAttributes = new[] { "System.Reflection.AssemblyMetadataAttribute" }
```
Comment (danielmarbach) [inline src/ServiceControl.UnitTests/API/APIApprovals.cs:29]: I'm aware that we previously did not remove the target framework either but I'm wondering why that would need to be in the approval file

**Comment by danielmarbach** [comment]
I reforced the CLA check that was stuck

File: src/ServiceControl.UnitTests/API/APIApprovals.cs (hunk 2 above)
Comment (SeanFeldman) [reply src/ServiceControl.UnitTests/API/APIApprovals.cs:29]: This TF is not modifying _what_ is verified. We strictly stick to fixing the issue of OublicApiApprovals.

**Comment by SeanFeldman** [comment]
Thank you, @danielmarbach .

**Pull request merged**